sdk/storage/storage-blob/src/TelemetryPolicyFactory.ts

```diff
@@ -37,6 +37,8 @@ export class TelemetryPolicyFactory implements RequestPolicyFactory {
 
     if (isNode) {
       if (telemetry) {
+        // FIXME: replace() only replaces the first space. And we have no idea why we need to replace spaces in the first place.
```

Comment:
Would better confirm with Xiaoning on this initial design.

Reply:
Already discussed about this.